### ISSUE TITLE
Small things to correct/enhance

### ISSUE THREAD

**Issue by marina-ricci** [LSSTDESC/CLMM#366]
Documentation :
1. - [ ] in clmm.theory.compute_* => "z_src_model" keyword : not clear from the documentation what it is doing.
 
. Enhancement : 
1. - [x] in clmm.theory.compute_* =>  allow to write either "NFW" or "nfw" (#370 )
 3.  - [ ] clmm.GalaxyCluster.make_radial_profile : we should be able to chose the error model : in case the noise is not shape noise dominated the errors shouldn't be computed as std/np.sqrt(N_src)
 
Bugs ?:
1. - [x] in example notebooks : we say the theory module works with "h" where it is not the case (done in #345 - no need for a new branch for this small change)
 5.  - [x] clmm.support.mock data not accessible from clmm.support need to import it explicitly (done in #368 )
 6.  - [x] when accessing e.g. cl.make_radial_profile we can not access the function documentation

**Comment by combet** [comment]
Thanks for this @marina-ricci!
- We have an open PR for the documentation (#365), so the first point can be taken care of in that branch and for v1.0. If you could take care of that'd be great.
- I'd suggest we leave the enhancements as such for the time being. We mention in the paper, below eq 12 that we will improve on this soon (that's clearly a limitation as of now)
- We should definitely fix any mention of 'h' in the notebooks for v1.0. Could you also take care of it?

**Comment by marina-ricci** [comment]
OK. 
What about points 2 and 3 of "Bugs".
`clmm/support/__init__.py` is empty. I guess this causes point 2.

**Comment by m-aguena** [comment]
I created a quick fix for `clmm.support.mock data not accessible from clmm.support need to import it explicitly` - PR #368 

@marina-ricci I do not see this error `when accessing e.g. cl.make_radial_profile we can not access the function documentation`. How are you getting that?

**Comment by marina-ricci** [comment]
Thank's @m-aguena!
- I just checked and (3) is quite subtle. 

> 

> `test = [clmm.GalaxyCluster('mock', cluster_ra, cluster_dec, 
                               cluster_z, mock_catalogs[i]), clmm.GalaxyCluster('mock', cluster_ra, cluster_dec, 
                               cluster_z, mock_catalogs[i])]

test1 = clmm.GalaxyCluster('mock', cluster_ra, cluster_dec, 
                               cluster_z, mock_catalogs[i])`

**Comment by m-aguena** [comment]
@marina-ricci I am not sure the reason, but this syntax does not work.
This is not a `clmm` problem. For instance, try:
```
import numpy as np
[np][0].array?
```
and you will get the same error.

My suggestion is to use either
```
x = test[0]
x.make_radial_profile?
```
or
```
help(test[0].make_radial_profile)
```

**Comment by m-aguena** [comment]
@marina-ricci I added a quick fix for enhancement 1 proposed in PR #370 .
@combet do you think we can put this in v1.0?

**Comment by combet** [comment]
If it's ready to go and doesn't break anything - sure ! Thanks @m-aguena for implementing and @marina-ricci for the suggestion. Will look at the PR tomorrow.
